docs/optimization.rst

```diff
@@ -308,6 +306,27 @@ cutting latency in half. The benefit provided by TensorRT will vary
 based on the model, but in general it can provide significant
 performance improvement.
 
+TensorFlow Automatic FP16 Optimization
+......................................
+
+TensorFlow has another option to provide FP16 optimization that can be
```

Comment:
provide speedup by using the FP16

Reply:
I don't want to say "speedup" explicitly. Already calling this an optimization.

Reply:
Makes sense. Resolving